code/game/dna/mutations/powers.dm

```diff
@@ -216,6 +216,7 @@
 	..()
 	M.alpha = 255
 	M.invisibility = 0
+	M.add_to_all_human_data_huds()
```

Comment:
Would perhaps be smart to make a setter for alpha and invisibility values.
Somebody will forget to call this proc if they ever set a mob to be invisible/visible
`make_invisible` with `make_visible` perhaps? Since it seems all paths here set the alpha to 128 if the mob becomes invisible